apps/docs/content/guides/api/automatic-retries-in-supabase-js.mdx

```diff
@@ -0,0 +1,118 @@
+---
+id: automatic-retries-in-supabase-js
+title: 'How to do automatic retries with `supabase-js`'
+subtitle: 'Learn how to add automatic retries to your Supabase API requests using `fetch-retry`.'
+---
+
+<Admonition type="danger" title="Important">
+  - You should only enable retries if your requests fail with network errors (e.g. 520 status from
+  Cloudflare) - A high number of retries have the potential to exhaust the Data API connection pool,
```

Comment:
Bulletpoints are not rendering well on the preview:
```suggestion
  Cloudflare) 
  - A high number of retries have the potential to exhaust the Data API connection pool,
```

Reply:
The `Admonition` component doesn't render bulletpoints